lab_1_keywords_tfidf/main.py

```diff
@@ -48,7 +66,19 @@ def calculate_frequencies(tokens: list[str]) -> Optional[dict[str, int]]:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not len(tokens) != 0 and not isinstance(tokens, list):
+        return None
+    else:
```

Comment:
we do not need else if the previous branch ends with return

Reply:
but when I don't use "else", then the next line gets highlighted and the code doesn't work

Reply:
now works, thanks